[2.9 ACRA] crash while handling search request with no results?

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/report-details/8d2619a3-e9f6-418d-99ad-badd6c3b3993

```
java.lang.RuntimeException: An error occurred while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:353)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
at java.util.concurrent.FutureTask.run(FutureTask.java:271)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
at java.lang.Thread.run(Thread.java:764)
Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'int java.util.List.size()' on a null object reference
at com.ichi2.async.DeckTask.doInBackgroundSearchCards(DeckTask.java:8)
at com.ichi2.async.DeckTask.doInBackground(DeckTask.java:36)
at com.ichi2.async.DeckTask.doInBackground(DeckTask.java:1)
at android.os.AsyncTask$2.call(AsyncTask.java:333)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
```

--- a/AnkiDroid/src/main/java/com/ichi2/libanki/Finder.java
+++ b/AnkiDroid/src/main/java/com/ichi2/libanki/Finder.java
@@ -1069,7 +1069,7 @@ public class Finder {
                 // cancel if the launching task was cancelled. 
                 if (task.isCancelled()){
                     Timber.i("_findCardsForCardBrowser() cancelled...");
-                    return null;
+                    return new ArrayList<>();
                 }                
                 Map<String, String> map = new HashMap<>();
                 map.put("id", cur.getString(0));

